lib/events/api.go

```diff
@@ -764,6 +764,8 @@ const (
 	ClusterNetworkingConfigUpdateEvent = "cluster_networking_config.update"
 	// SessionRecordingConfigUpdateEvent is emitted when a user updates the cluster session recording configuration.
 	SessionRecordingConfigUpdateEvent = "session_recording_config.update"
+	// AccessGraphSettingsUpdateEvent is emitted when a user updates the access graph settings configuration.
```

Comment:
The confusing part about it is that we will have multiple Access Graph setting in Teleport now. Saying "AG config has been updated" doesn't indicate which configuration.

Reply:
I can include the config change but it won't render well